xamarin/Xamarin.Forms
# [iOS] Fixed ListView cannot reset selected status

### Description of Change ###
Added _lastSelected variable in listview renderer to support unselect action

### Issues Resolved ###
fixes #4394 

### API Changes ###
None

### Platforms Affected ###
iOS

### Behavioral/Visual Changes ###
In iOS, now you can deselect the last selected item in the list view.

### Before/After Screenshots ###
## Before ##
![before](https://user-images.githubusercontent.com/18006944/49396812-e85b6f80-f79e-11e8-98ae-82e27289eb51.gif)

## After ##
![after](https://user-images.githubusercontent.com/18006944/49396823-ec878d00-f79e-11e8-85f8-3f463852b7d5.gif)

### Testing Procedure ###
Start ControlGallery.iOS, Search and go to G4394 Page in tests screen, you should be able to unselect the selected item now
